paasta_tools/kubernetes_tools.py

```diff
@@ -3526,3 +3531,60 @@ def get_kubernetes_secret_env_variables(
                 kube_client, secret_name, service_name
             )
     return decrypted_secrets
+
+
+def get_kubernetes_secret_volumes(
+    kube_client: KubeClient,
+    secret_volumes_config: Dict[str, Any],
+    service_name: str,
+) -> Dict[str, str]:
```

Comment:
should this be `Dict[str, Union[str, bytes]]` to account for binary file contents?

(same in `decrypt_secret_volumes()` and `decrypt_secret()` if this does need to be updated)

Reply:
Fixed the mypy type hints